packages/@aws-cdk/cdk/lib/stack.ts

```diff
@@ -31,15 +33,16 @@ export interface StackProps {
   readonly namingScheme?: IAddressingScheme;
 
   /**
-   * Should the Stack be deployed when running `cdk deploy` without arguments
-   * (and listed when running `cdk synth` without arguments).
-   * Setting this to `false` is useful when you have a Stack in your CDK app
-   * that you don't want to deploy using the CDK toolkit -
-   * for example, because you're planning on deploying it through CodePipeline.
+   * Indicates if this stack is hidden, which means that it won't be
```

Comment:
`hidden` does not imply _not automatically deployed_ to me. It would kind of imply _not visible in `cdk ls`_... The semantics are getting very blurry here, which I don't like.

Reply:
+1. I'm also not entirely sure what was wrong with autoDeploy.

Reply:
Removed this feature